### ISSUE TITLE
bug: The client only once automatically exits fullscreen mode

### ISSUE THREAD

**Issue by GucekPL** [YT-Advanced/ReX-patches#62]
## Bug description
When I enter a random video after starting the app and want to switch to fullscreen mode, the app automatically takes itself out of fullscreen mode. This is only the first time I start the video after starting the app - after that, there is no problem

## Steps to reproduce
1. Start app
2. Press random video
3. Go to fullscreen mode and wait when app automatically close fullscreen

## Video

https://github.com/YT-Advanced/ReX-patches/assets/69505731/c582f470-28c7-4045-b505-273ceed67f15

### Additional context

**Device**: HUAWEI JNY-LX1
**Patches version**: 2.220.4
**Patching method**: RVX Manager 1.18.1
**YT Version**: 19.04.37

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by YT-Advanced** [comment]
This bug seem not reproduced in my side, can you share your settings

**Comment by GucekPL** [comment]
```"bypass_ambient_mode_restrictions": true,
"close_interstitial_ads": true,
"custom_playback_speeds": "0.25\n0.5\n0.75\n1.0\n1.25\n1.5\n1.75\n2.0\n2.25\n2.5\n2.75\n3.0",
"default_playback_speed": 1,
"default_video_quality_mobile": 1080,
"default_video_quality_wifi": 1080,
"disable_auto_captions": true,
"disable_speed_overlay": true,
"enable_custom_playback_speed": true,
"enable_opus_codec": true,
"enable_save_video_quality": false,
"enable_swipe_brightness": false,
"enable_swipe_volume": false,
"enable_video_codec": true,
"enable_video_codec_type": true,
"hide_account_menu": true,
"hide_account_menu_filter_strings": "Przejdź na YouTube Premium\nKorzyści, jakie daje Premium\nPobrane\nCzas oglądania\nZakupy i subskrypcje\nTwoje dane w YouTube\nPomoc i opinie\nJak działa YouTube\nYouTube Studio\nYouTube Music\nYouTube Kids\nYouTube TV\nTwoje filmy\nKupione filmy",
"hide_audio_track_button": false,
"hide_autoplay_preview": true,
"hide_button_create_clip": true,
"hide_button_download": true,
"hide_button_remix": true,
"hide_button_report": true,
"hide_button_rewards": true,
"hide_button_shop": true,
"hide_button_thanks": true,
"hide_cast_button": false,
"hide_comments_thanks_button": true,
"hide_create_shorts_button": true,
"hide_emoji_picker": true,
"hide_end_screen_overlay": true,
"hide_feed_flyout_panel": true,
"hide_feed_flyout_panel_filter_strings": "Odtwórz jako następny w kolejce\nZapisz na liście Do obejrzenia\nPobierz film\nZgłoś",
"hide_fullscreen_panels": true,
"hide_game_section": true,
"hide_home_feed_membership_video": true,
"hide_info_cards_section": true,
"hide_mix_playlists": true,
"hide_place_section": true,
"hide_player_flyout_panel_audio_track": true,
"hide_player_flyout_panel_captions_footer": true,
"hide_player_flyout_panel_lock_screen": true,
"hide_player_flyout_panel_quality_footer": true,
"hide_seek_message": true,
"hide_seek_undo_message": true,
"hide_shorts_toolbar_banner": true,
"hide_shorts_toolbar_camera_button": true,
"hide_shorts_toolbar_menu_button": true,
"hide_shorts_toolbar_search_button": true,
"hide_suggested_video_overlay": true,
"hide_suggested_video_overlay_auto_play": false,
"hide_transcript_section": true,
"hide_youtube_music_button": true,
"remove_viewer_discretion_dialog": true,
"spoof_player_parameter_in_feed": true,
"swipe_brightness_value": -1,
"switching_create_notification": false,
"ryd_toast_on_connection_error": true,
"sb_toast_on_connection_error": true,
"sb_interaction": "seekbar-only",
"sb_poi_highlight": "seekbar-only",
"sb_intro": "seekbar-only",
"sb_outro": "seekbar-only",
"sb_preview": "seekbar-only",
"sb_filler": "seekbar-only",
"sb_music_offtopic": "seekbar-only"witching_create_notification": false,
"sb_local_time_saved_milliseconds": 138306,
"sb_local_time_saved_number_segments": 7,
"sb_interaction": "seekbar-only",
"sb_poi_highlight": "seekbar-only",
"sb_intro": "seekbar-only",
"sb_outro": "seekbar-only",
"sb_preview": "seekbar-only",
"sb_filler": "seekbar-only",
"sb_music_offtopic": "seekbar-only"highlight": "seekbar-only",
"sb_intro": "seekbar-only",
"sb_outro": "seekbar-only",
"sb_preview": "seekbar-only",
"sb_filler": "seekbar-only",
"sb_music_offtopic": "seekbar-only"```

**Comment by GucekPL** [comment]
Yesterday I Installed the same APK on friend's phone and she hasn't this problem. After that I tried clean install, and problem was already after install
